client/src/components/SwipeUpFooter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import SwipeArrowSrc from "../assets/swipeArrow.svg";
+import styled from "@emotion/styled";
+import { Link } from "react-router-dom";
+
+const SwipeFooter = styled.footer`
+  height: 2rem;
+  width: 100%;
+  background: rgba(134, 134, 134, 0.5);
```

Comment:
You could use CSS variables to store your colors and give them names, if you like.